addon/components/inputs/date.js

```diff
@@ -0,0 +1,28 @@
+import AbstractInput from './abstract-input'
+import layout from 'ember-frost-bunsen/templates/components/frost-bunsen-input-date'
+
+export default AbstractInput.extend({
+  // == Component Properties ===================================================
+
+  classNameBindings: ['value:frost-bunsen-has-value'],
+
+  classNames: [
+    'frost-bunsen-input-date',
+    'frost-field'
+  ],
+
+  layout,
+
+  // == Computed Properties ====================================================
+
+  // == Functions ==============================================================
+
+  // == Actions ===============================================================
+
+  actions: {
+    selectDate (pikaday) {
```

Comment:
Add JSDoc please, I have no idea what the format of `pikaday` is. String? Object? If so, waht keys/values?